GOVERNANCE.md

```diff
@@ -1,18 +1,100 @@
 ---
 # SPDX-License-Identifier: CC0-1.0
-# SPDX-FileCopyrightText: 2019-2024 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS
+# SPDX-FileCopyrightText: 2024 Standard for Public Code Authors, https://github.com/standard-for-public-code/standard-for-public-code/blob/develop/AUTHORS.md
```

Comment:
Maybe the year can even be dropped here as some of the authors have copyrights going back.

Reply:
I agree